eiffel-vocabulary/EiffelServiceResumedEvent.md

```diff
@@ -0,0 +1,20 @@
+
+# EiffelSeriveResumedEvent (ArtS)
+
+The EiffelServiceResumedEvent declares that a service has been Resumed and is ready for use.
```

Comment:
Perhaps be a bit more descriptive here, explaining the relationship to ServicePaused and ServiceStarted? Being more verbose in the event descriptions is a general comments for all event types.